### DISCUSSION TITLE
Video length

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/RAG_Hack#80] [Q&A] [answered]
I will try to make the video smaller, but is it okay if the length of the video ends up being a bit longer than 5 minutes, perhaps by 1 to 4 minutes?

I would like to make some intro, how the app can be used in real world scenario, and in which business cases. And then briefly show all features, and a bit of code and prompts of main workflow.

**Comment by multispark** [comment, marked as answer]
Hello Yuri, we have a time limit on the video to help keep judging sustainable! Please try to stay as close to the 5 minutes as possible!
